src/goto-symex/symex_assign.cpp

```diff
@@ -231,10 +232,108 @@ void goto_symext::symex_assign_symbol(
     tmp_ssa_rhs.swap(ssa_rhs);
   }
 
+  // TODO: done via rename
+  // field_sensitivityt::apply(ns, ssa_rhs, false);
```

Comment:
The TODO comment is not clear (what has to be done?). Ideally it should be resolved before the PR is merged.

Reply:
Remove, actually the call to `rename` is much clearer now as it takes an additional `field_sensitivity` parameter.